packages/sku/src/services/vite/loadable/helpers/preloadUtils.ts

```diff
@@ -14,14 +14,14 @@ const getNonce = (nonce?: string) => (nonce ? ` nonce="${nonce}"` : '');
 
 const tagTypes = {
   stylesheet: ({ href, nonce }: Preload) =>
-    `<link rel="stylesheet" href="/${href}" crossorigin${getNonce(nonce)} />`,
+    `<link rel="stylesheet" href="./${href}" crossorigin${getNonce(nonce)} />`,
```

Comment:
This is sort of replacing one problem with another. Now any href is always relative.

We'd ideally support any type of path, e.g. all of these should work: `/static/foo`, `./`, `/`. I think with the suggestion in my other comment, combined with making all of these just `href="${href}", any `publicPath` to work.